### PR TITLE
Fixup README for Just the Docs documentation website

### PR DESCRIPTION
**Pull request by pniedzielski**
*Issue number of the reported bug or feature request:* #106

**Describe your changes**
These two patches improve the Just the Docs documentation website README, by clarifying the license notice for our use of the Just the Docs Jekyll theme and by removing stray references to tools that most users may not have access to.

**Testing performed**
As this is just the `README`, I ensured that the Markdown file renders correctly, and that all tools referenced are available to most end users.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -4,15 +4,21 @@ Use [Just the Docs](https://github.com/just-the-docs/just-the-docs) for document
 
 ## License
 
-Uses [Just the Docs](https://github.com/just-the-docs/just-the-docs)
+In order to build the documentation, we use the [Just the
+Docs](https://github.com/just-the-docs/just-the-docs) theme for Jekyll, which
+is licensed under the [MIT license](LICENSE.just-the-docs.txt).  Please see
+that file for the theme’s copyright and permission notices.
```
Comment (kaikulimu) [inline docs/README.md:10]: Is it weird that `LICENSE.just-the-docs.txt` is under the `docs` directory instead of the `licenses` directory?

Comment (kaikulimu) [reply docs/README.md:10]: If we move it to the `licenses` directory, the file should be renamed `LICENSE-just-the-docs.txt` to be consistent with similar files.

**Review by kaikulimu — COMMENTED** [review]
Minor comments

File: docs/README.md (hunk 1 above)
Comment (pniedzielski) [reply docs/README.md:10]: I'll double-check to see whether we link to it from anywhere in the documentation (and thus, whether it needs to be where it is), and if not, I'll move it to `licenses` as you suggest.

Comment (pniedzielski) [reply docs/README.md]: Moved and renamed the license, changed the link to it in the documentation `README`, and removed the old path from the Jekyll excludes.

**Review by kaikulimu — APPROVED** [review]
LGTM

**Pull request merged**